### ISSUE TITLE
excalidraw version

### ISSUE THREAD

**Issue by LuciusPertis** [LuciusPertis/FAST-SRS#3]
we will use [excalidraw](https://github.com/excalidraw/excalidraw) to develop this ...

main OBJ :+1: 
- to make docs and written feature/requirements
- to extract diagrams (class, activity etc.)
- (future NOICE SHIT) repo-generating from SRS

**Comment by LuciusPertis** [comment]
- using excalidraw does achieve the OBJ for a standalone simple module (just some .json extraction is needed thereafter)
- complexity starts when modules/separate files need to integrate btw themselves, creation of consistent redundancies (for a single module across multiple repos/files/modules etc.

we will start this with dummy impl then increase inter-mod-integration little by little
(plz do this stuff dude wtf; YOUR IN UR FOURTH FUCKLIN YEAR)

 - [ ] create a simple dfd generation
 - [ ] CI/CD tests and req calls/database
 - [ ] inter-mod-integration

Basically: use excalidraw as frontend and data-def and build CI/CD backend
